digitalstrategie/assets/scss/_variables.scss

```diff
@@ -17,7 +17,7 @@ $green:   #0e4c38 !default;
 $teal:    #20c997 !default;
 $cyan:    #0dcaf0 !default;
 $white:   #ffffff !default; // added
-$gray:    #545454 !default; // added
+$gray:    #dddddd !default; // added
```

Comment:
i changed this to the same value as specified in the design. Since it is not used anywhere and there is no real alternative gray i changed it here.